### ISSUE TITLE
Can't launch

### ISSUE THREAD

**Issue by Cyliann** [FlareSolverr/FlareSolverr#942]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.3.6
- Last working FlareSolverr version: none
- Operating system: ARch Linux
- Are you using Docker: [yes/no] no
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no] no
- Are you using a Proxy: [yes/no] no
- Are you using Captcha Solver: [yes/no] no
- If using captcha solver, which one:
- URL to test this issue:
```

### Description

When I try to luanch Flaresolverr it immadiately stops with `Chrome / Chromium version not detected!`.
Systemd process also stops with:
```systemctl status flaresolverr.service
● flaresolverr.service - FlareSolverr
     Loaded: loaded (/usr/lib/systemd/system/flaresolverr.service; enabled; preset: disabled)
     Active: activating (auto-restart) (Result: exit-code) since Sat 2023-11-04 13:26:34 CET; 540ms ago
    Process: 30027 ExecStart=/opt/flaresolverr/flaresolverr (code=exited, status=1/FAILURE)
   Main PID: 30027 (code=exited, status=1/FAILURE)
        CPU: 329ms
```

### Logged Error Messages

```text
/opt/flaresolverr/flaresolverr
2023-11-04 13:04:58 INFO     FlareSolverr 3.3.6
2023-11-04 13:04:58 INFO     Testing web browser installation...
2023-11-04 13:04:58 INFO     Platform: Linux-6.5.9-arch2-1-x86_64-with-glibc2.38
2023-11-04 13:04:58 INFO     Chrome / Chromium path: /opt/flaresolverr/chrome/chrome
/bin/sh: symbol lookup error: /bin/sh: undefined symbol: rl_trim_arg_from_keyseq
2023-11-04 13:04:58 ERROR    Chrome / Chromium version not detected!
```

**Comment by Brokeos** [comment]
Exactly same issue

```
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 105, in <module>
    flaresolverr_service.test_browser_installation()
  File "/app/flaresolverr_service.py", line 72, in test_browser_installation
    user_agent = utils.get_user_agent()
                 ^^^^^^^^^^^^^^^^^^^^^^
  File "/app/utils.py", line 300, in get_user_agent
    raise Exception("Error getting browser User-Agent. " + str(e))
Exception: Error getting browser User-Agent. Message: session not created
from unknown error: result.webdriverValue.value list is missing or empty in Runtime.callFunctionOn response
  (Session info: chrome=119.0.6045.105)
Stacktrace:
#0 0x5615fe4d0023 <unknown>
#1 0x5615fe1ec9a8 <unknown>
#2 0x5615fe1f14c5 <unknown>
#3 0x5615fe1f2bb2 <unknown>
#4 0x5615fe1f2ca4 <unknown>
#5 0x5615fe25e50a <unknown>
#6 0x5615fe254bb3 <unknown>
#7 0x5615fe21f9de <unknown>
#8 0x5615fe220d72 <unknown>
#9 0x5615fe49efb6 <unknown>
#10 0x5615fe4a202d <unknown>
#11 0x5615fe4a1ac6 <unknown>
#12 0x5615fe4a24d5 <unknown>
#13 0x5615fe4a890b <unknown>
#14 0x5615fe4a2856 <unknown>
#15 0x5615fe47bb0d <unknown>
#16 0x5615fe4bb265 <unknown>
#17 0x5615fe4bb45f <unknown>
#18 0x5615fe4c904f <unknown>
#19 0x7fc03b984ea7 start_thread
```

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/928

**Comment by Cyliann** [comment]
Something is wrong with `flaresolverr-bin` in the AUR. For me installing Flaresolverr from source fixed the problem

**Comment by ilike2burnthing** [comment]
@txtsd can you update to v3.3.7? Thanks.

**Comment by txtsd** [comment]
I'll do it today.

Idk how building from source worked though since the flaresolverr and flaresolverr-git packages depend on system selenium which is a higher version than the one that works.

**Comment by txtsd** [comment]
AUR packages have been updated.

**Comment by ilike2burnthing** [comment]
Thanks.

@Cyliann / @Brokeos any luck with v3.3.7?

**Comment by Cyliann** [comment]
> I'll do it today.
> 
> Idk how building from source worked though since the flaresolverr depends on system selenium which is a higher version than the one that works.

So the thing was, my issue wasn't really with selenium but with absence of Chromium (or so I think), when I built from source I installed Chromium manually and flaresolverr that problem was gone. But then I had the same problem with selenium as others. 

Updated to 3.3.7 and the issue persists

**Comment by ilike2burnthing** [comment]
> If not, what version of Chrome/Chromium are you using?

**Comment by Cyliann** [comment]
> > If not, what version of Chrome/Chromium are you using?

Sorry, didn't read that one
Chromium version: 119.0.6045.105

**Comment by txtsd** [comment]
OPs issue stems from having an older readline in the package. I fixed it in `flaresolverr-bin` by removing libreadline.

**Comment by txtsd** [comment]
@Cyliann
 https://github.com/FlareSolverr/FlareSolverr/issues/928#issuecomment-1784000384

You can also downgrade by editing the `pkgver` in the `PKGBUILD` to that version, and then building it.

**Comment by Cyliann** [comment]
> You can also downgrade by editing the `pkgver` in the `PKGBUILD` of `python-selenium` to that version, and then building it.

Thanks. I didn't know that. 
I downgraded `python-selenium`, but now I'm getting this:
```
2023-11-07 17:17:25 INFO     FlareSolverr 3.3.7
2023-11-07 17:17:25 INFO     Testing web browser installation...
2023-11-07 17:17:25 INFO     Platform: Linux-6.5.9-arch2-1-x86_64-with-glibc2.38
2023-11-07 17:17:25 INFO     Chrome / Chromium path: /usr/bin/chromium
2023-11-07 17:17:26 INFO     Chrome / Chromium major version: 119
2023-11-07 17:17:26 INFO     Launching web browser...
Traceback (most recent call last):
  File "/opt/flaresolverr/utils.py", line 296, in get_user_agent
    driver = get_webdriver()
             ^^^^^^^^^^^^^^^
  File "/opt/flaresolverr/utils.py", line 177, in get_webdriver
    driver = uc.Chrome(options=options, browser_executable_path=browser_executable_path,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/flaresolverr/undetected_chromedriver/__init__.py", line 474, in __init__
    super(Chrome, self).__init__(
  File "/usr/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/usr/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py", line 56, in __init__
    super().__init__(
  File "/usr/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 205, in __init__
    self.start_session(capabilities)
  File "/opt/flaresolverr/undetected_chromedriver/__init__.py", line 732, in start_session
    super(selenium.webdriver.chrome.webdriver.WebDriver, self).start_session(
  File "/usr/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 289, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 344, in execute
    self.error_handler.check_response(response)
  File "/usr/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created
from unknown error: result.webdriverValue.value list is missing or empty in Runtime.callFunctionOn response
  (Session info: chrome=119.0.6045.105)
Stacktrace:
#0 0x561b148454e3 <unknown>
#1 0x561b14574c76 <unknown>
#2 0x561b14578d50 <unknown>
#3 0x561b1457a97e <unknown>
#4 0x561b1457aa4c <unknown>
#5 0x561b145d9f77 <unknown>
#6 0x561b145d847f <unknown>
#7 0x561b145cfde3 <unknown>
#8 0x561b145a52dd <unknown>
#9 0x561b145a634e <unknown>
#10 0x561b148053e4 <unknown>
#11 0x561b148093d7 <unknown>
#12 0x561b14813b20 <unknown>
#13 0x561b1480a023 <unknown>
#14 0x561b147d81aa <unknown>
#15 0x561b1482e6b8 <unknown>
#16 0x561b1482e847 <unknown>
#17 0x561b1483e243 <unknown>
#18 0x7f92b3da49eb <unknown>


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/flaresolverr/flaresolverr.py", line 105, in <module>
    flaresolverr_service.test_browser_installation()
  File "/opt/flaresolverr/flaresolverr_service.py", line 72, in test_browser_installation
    user_agent = utils.get_user_agent()
                 ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/flaresolverr/utils.py", line 300, in get_user_agent
    raise Exception("Error getting browser User-Agent. " + str(e))
Exception: Error getting browser User-Agent. Message: session not created
from unknown error: result.webdriverValue.value list is missing or empty in Runtime.callFunctionOn response
  (Session info: chrome=119.0.6045.105)
Stacktrace:
#0 0x561b148454e3 <unknown>
#1 0x561b14574c76 <unknown>
#2 0x561b14578d50 <unknown>
#3 0x561b1457a97e <unknown>
#4 0x561b1457aa4c <unknown>
#5 0x561b145d9f77 <unknown>
#6 0x561b145d847f <unknown>
#7 0x561b145cfde3 <unknown>
#8 0x561b145a52dd <unknown>
#9 0x561b145a634e <unknown>
#10 0x561b148053e4 <unknown>
#11 0x561b148093d7 <unknown>
#12 0x561b14813b20 <unknown>
#13 0x561b1480a023 <unknown>
#14 0x561b147d81aa <unknown>
#15 0x561b1482e6b8 <unknown>
#16 0x561b1482e847 <unknown>
#17 0x561b1483e243 <unknown>
#18 0x7f92b3da49eb <unknown>
```

**Comment by ilike2burnthing** [comment]
Seeing this elsewhere with v119:
- https://github.com/FlareSolverr/FlareSolverr/issues/936#issuecomment-1796859801
- https://github.com/FlareSolverr/FlareSolverr/issues/945

This was first seen with Docker, but rebuilding resolved it (hence v3.3.7):
- https://github.com/FlareSolverr/FlareSolverr/issues/880#issuecomment-1793413995

I'm assuming it's a dependency issue, but I have no idea what.

**Comment by Brokeos** [comment]
> Thanks.
> 
> @Cyliann / @Brokeos any luck with v3.3.7? If not, what version of Chrome/Chromium are you using?

3.3.7 seems to work for me (jackett restarted)
Chromium 119.0.6045.105 built on Debian 11.8, running on Debian 11.8
